test/feature_flags_manager_test.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:instabug_flutter/src/generated/instabug.api.g.dart';
+import 'package:instabug_flutter/src/utils/feature_flags_manager.dart';
+import 'package:instabug_flutter/src/utils/ibg_build_info.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'feature_flags_manager_test.mocks.dart';
+
+@GenerateMocks([InstabugHostApi, IBGBuildInfo])
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  WidgetsFlutterBinding.ensureInitialized();
+
+  final mInstabugHost = MockInstabugHostApi();
+  final mBuildInfo = MockIBGBuildInfo();
+
+  setUpAll(() {
+    FeatureFlagsManager.$setHostApi(mInstabugHost);
+    IBGBuildInfo.setInstance(mBuildInfo);
+  });
+
+  tearDown(() {
+    reset(mInstabugHost);
+  });
+
+  test('[isW3ExternalTraceID] should call host method on IOS', () async {
```

Comment:
Let's use the actual name and change W3* members to W3C*  for all of the file content.